### ISSUE TITLE
H74 Need Clarification on requirement for "complete start and end tags" under 4.1.1 Parsing

### ISSUE THREAD

**Issue by jgauvreau** [w3c/wcag#306]
Regarding starting element tags, the WCAG 2.0 4.1.1 Parsing success criterion requires "elements have complete start and end tags, elements are nested according to their specifications". 
WCAG 2.0 Technique H74: Ensuring that opening and closing tags are used according to specification states "This technique deals only with errors related to incorrectly formed opening and closing tags". 

We are seeking clarification on what the term "complete" means in this statement as well as the test procedure for technique H74. Recently, when using the Nu HTML Checker to perform validation on HTML documents we have seen errors that state something like "Attribute XXX not allowed on element YYY at this point". 
1) Would this "attribute not allowed on element" error be considered a failure to meet the WCAG 2.0 4.1.1 Parsing requirement related to the requirement that elements have "complete" start tags
2) Or is this an HTML validation error that is not a strict requirement under 4.1.1 and should be filtered out of HTML validation results, perhaps through creation of a customized version of the "Filter Nu Validator Results" bookmarklet in the Web Accessibility Toolbar (WAT).

**Comment by patrickhlauke** [comment]
"complete" means that, unless an element is self-closing (like `<br>`, `<img>`, `<hr>` etc), there needs to be an opening and a matching closing tag.

while non-standard/disallowed attributes present validation errors, they should generally not have any accessibility impact (?) so that should be ignorable.

see also https://developer.paciellogroup.com/blog/2015/11/wcag-2-0-parsing-criterion-is-a-pita/

In addition, I'd note that modern user agents should consistently parse malformed/wrongly closed tags, duplicate attributes, etc as they should follow the more detailed HTML5 parsing algorithm. So in theory, even badly malformed markup should at least be turned into the same DOM...

**Comment by jgauvreau** [comment]
Thanks for replying so quickly last night, Patrick! Your interpretation is what my team and I were leaning towards but I wanted to get a second opinion. Thanks, again.
